llvm/test/tools/llvm-readobj/ELF/dynamic-tags.test

```diff
@@ -8,6 +8,9 @@
 # RUN: llvm-readelf --dynamic-table %t1 \
 # RUN:   | FileCheck %s --strict-whitespace --match-full-lines --check-prefix=GNU64
 # RUN: llvm-readelf -d %t1 | FileCheck %s --check-prefix=GNU64
+# RUN: llvm-readelf --dynamic-table --pretty-print --elf-output-style=JSON %t1 \
+# RUN:   | FileCheck %s --check-prefix=JSON64
```

Comment:
Could you check what other JSON output testing does, please as to whether it uses `--match-full-lines` and `--strict-whitespace`? I suspect the majority don't, so nothing to change here, but some minor changes can be made to the output test patterns (see below).

Reply:
So I added --strict-whitespace back because once I fixed the indenting that passed fine.